src/components/BlogPost.jsx

```diff
@@ -11,7 +11,8 @@ function BlogPost({ author, title, excerpt }) {
           <img
             className="authorImage"
             src={`https://joeschmoe.io/api/v1/${author}`}
-            alt="Author"
+            alt={`${author} avatar`}
+            title={author}
```

Comment:
These changes are not related, revert them.

Reply:
reverted